src/RestApi/StoreApi/Controllers/CartShippingRates.php

```diff
@@ -121,8 +121,9 @@ function( $item ) {
 		$packages = $this->get_shipping_packages( $request );
 		$response = [];
 
-		foreach ( $packages as $package ) {
-			$response[] = $this->prepare_response_for_collection( $this->prepare_item_for_response( $package, $request ) );
+		foreach ( $packages as $key => $package ) {
+			$package['key'] = $key;
```

Comment:
In 3ddf8b70a5750c1d2c7a02a054089c90ae6bfdb5 I renamed this from `index` to `key`, because its values might not always be ordered integers.